scripts/example.ts

```diff
@@ -0,0 +1,144 @@
+import path from 'path';
+import { remove, pathExists } from 'fs-extra';
+import prompts from 'prompts';
+
+import { getOptionsOrPrompt } from './utils/options';
+import type { CLIStep } from './utils/cli-step';
+import { executeCLIStep } from './utils/cli-step';
+
+const frameworks = ['react'];
+const addons = ['a11y', 'storysource'];
+const examplesDir = path.resolve(__dirname, '../examples');
+
+async function getOptions() {
+  return getOptionsOrPrompt('yarn example', {
+    framework: {
+      description: 'Which framework would you like to use?',
+      values: frameworks,
+      required: true,
+    },
+    addon: {
+      description: 'Which extra addons (beyond the CLI defaults) would you like installed?',
+      values: addons,
+      multiple: true,
+    },
+    includeStories: {
+      description:
+        "Include Storybook's own stories (only applies if a react-based framework is used)?",
```

Comment:
This should probably be part of a validation from previous prompts, if possible. So that this question only is prompted if it makes sense to be prompted. This way the amount of prompting becomes smaller

Reply:
Yep, I'll add a way of making prompts conditional.